spec/components/schemas.yaml

```diff
@@ -818,6 +818,150 @@ components:
       description: Feature identifier (unique per collection)
       example: smr200-200-4-2019
       type: string
+    itemGeometry:
+      oneOf:
+        - title: GeoJSON Point
+          type: object
+          required:
+            - type
+            - coordinates
+          properties:
+            type:
+              type: string
+              enum:
+                - Point
+            coordinates:
+              type: array
+              minItems: 2
+              items:
+                type: number
+            bbox:
+              type: array
+              minItems: 4
+              items:
+                type: number
```

Comment:
I would suggest to break down the schema for each object type in the `onOf`:

```yaml
components:
  schemas:
    itemGeometry:
      onOf:
        - $ref: "#/components/schemas/geoJsonPoint"
       ...
    geoJsonPoint:
      title: GeoJSON Point
       type: object
       required:
          - type
          - coordinates
       properties:
         type:
           type: string
           enum:
             - Point
         coordinates:
           type: array
           minItems: 2
           items:
             type: number
         bbox:
           type: array
           minItems: 4
           items:
             type: number
```
Then we can keep all GeoJSON schemas and only remove refs in `itemGeometry` if needed. This can be useful for future uses instead of removing all parts from the itemGeometry or commenting out.